testing/haxe/APKBUILD

```diff
@@ -13,9 +13,11 @@ makedepends="camlp4 ocaml neko-dev zlib-dev pcre-dev"
 options="!strip !makeflags"
 install=""
 subpackages="$pkgname-doc"
+_commit_haxelib="a494d8be523e26fcf875e2c33915808dc221e17a"
+_commit_ocamllibs="ab5be31c6dd1fcd761c2ba16c5d767bcf6792490"
```

Comment:
:+1: Good to have those in variables. It would be nice if you could add a comment that tells how to find the commit hashes.